atum/src/main/scala/za/co/absa/atum/core/Accumulator.scala

```diff
@@ -60,6 +60,12 @@ class Accumulator() {
     }
   }
 
+  /**
+   * Ability to view the storer if set.
+   * @return
+   */
+  private[atum] def getStorer: Option[ControlMeasuresStorer] = if (isStorerLoaded) Some(storer) else None
```

Comment:
This way, the accumulator insides are opened to the rest of the Atum to behave differently based on what kind of storer (HDFS vs S3) is used.